interface/ceed-qfunction.c

```diff
@@ -573,7 +573,7 @@ int CeedQFunctionGetFlopsEstimate(CeedQFunction qf, CeedSize *flops) {
   @param[in]  f          Function pointer to evaluate action at quadrature points.
                            See \ref CeedQFunctionUser.
   @param[in]  source     Absolute path to source of QFunction, "\abs_path\file.h:function_name".
-                           For support across all backends, this source must only contain constructs supported by C99, C++11, and CUDA.
+                           This source must only contain constructs supported by all targeted backends (i.e. CUDA for `/gpu/cuda`, OpenCL/SYCL for `/gpu/sycl`, etc.).
```

Comment:
Should update to clarify that the entire contents of the file will be JiTed and must stand alone with all local includes

Reply:
Done.